cmake/conan_setup.cmake

```diff
@@ -30,5 +33,6 @@ conan_cmake_run(
     OPTIONS build_benchmarks=${benchmark_option}
             build_tests=${tests_option}
     SETTINGS compiler.cppstd=17
+    ${compiler.libcxx}
```

Comment:
```suggestion
            compiler.libcxx=${compiler_libcxx}
```

Reply:
My intention is to not even pass the `SETTINGS`-argument to conan when we're not on `UNIX`. That would mean always passing that argument, wouldn't it?

Reply:
oh right, but still the underscore version?